inc/dg/Makefile

```diff
@@ -1,4 +1,4 @@
-device=gpu
+device=omp
```

Comment:
Should we change the default to omp? I don't care so much

Reply:
Ohh yeah I forgot thi change, this was only to be able to run to code. I doesn't matter to me to be :)

Reply:
Also, I'll make the change to documentation.